stacks-common/src/types/mod.rs

```diff
@@ -75,6 +75,7 @@ pub enum StacksEpochId {
     Epoch22 = 0x0200f,
     Epoch23 = 0x02014,
     Epoch24 = 0x02019,
+    Epoch25 = 0x0201e,
```

Comment:
The next epoch is going to be `Epoch30`.  If this PR is ready in time, it will ship with Nakamoto v1 (Stacks 3.0).